sorting/strand_sort.cpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * @file strand_sort.cpp
+ * @brief Implementation of [Strand Sort](https://en.wikipedia.org/wiki/Strand_sort) algorithm.
+ *
+ * @details
+ * Strand Sort is a sorting algorithm that works in (O(n)) time if list is already sorted and works in (O(n * n)) in worst case.
+ * 
+ * ### Algorithm
+ * + It is passed over the array to be sorted once and the ascending (sequential) numbers are taken.
```

Comment:
```suggestion
 * It is passed over the array to be sorted once and the ascending (sequential) numbers are taken.
```
1. please remove the `### Algorithm` as it is redundant and will be properly formatted by doxygen
2. please remove the preceding `+` on the lines. If implying bullet points, then use `*` or `-`